packages/modals/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@goodhood/modals",
+  "description": "Modal components",
+  "author": "goodhood.eu <max@nebenan.de>",
+  "license": "SEE LICENSE IN LICENSE FILE",
+  "homepage": "https://goodhood.eu/",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/goodhood-eu/goodhood.git"
+  },
+  "bugs": {
+    "url": "https://github.com/goodhood-eu/goodhood/issues"
+  },
+  "version": "0.0.1",
+  "module": "lib/index.esm.js",
+  "main": "lib/index.js",
+  "sideEffects": false,
+  "files": [
+    "/lib"
+  ],
+  "scripts": {
+    "reinstall": "rm -rf node_modules && yarn install",
+    "prepublishOnly": "(cd ../../ && npm run check) && npm run build",
+    "build": "rollup --config rollup.config.js",
+    "start": "../../scripts/start-preview",
+    "test": "../../scripts/run-tests"
+  },
+  "peerDependencies": {
```

Comment:
I know we did it different before but let's move some of those into `dependencies`

- clsx
- nebenan-helpers
- prop-types
- goodhood/icons

No need to have those in peer deps - only makes updating packages super hard

Reply:
oh man we have to have nebenan-helpers in peer deps because of keymanager ...